packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -229,6 +238,40 @@ export interface SetLoanToken {
   interest?: BigNumber
 }
 
+export interface ListLoanTokenResult {
+  [key: string]: LoanTokenDetails
+}
+
+export interface LoanTokenDetails {
+  token: LoanTokenData
+  priceFeedId: string
+  interest: BigNumber
+}
+
+export interface LoanTokenData {
+  [key: string]: LoanTokenInfo
+}
+
+export interface LoanTokenInfo {
+  collateralAddress: string
+  creationHeight: BigNumber
+  creationTx: string
+  decimal: BigNumber
+  destructionHeight: BigNumber
+  destructionTx: string
+  finalized: boolean
+  isDAT: boolean
+  isLPS: boolean
+  isLoanToken: boolean
+  limit: BigNumber
+  mintable: boolean
+  minted: BigNumber
+  name: string
+  symbol: string
+  symbolKey: string
+  tradeable: boolean
+}
```

Comment:
```suggestion
export interface LoanTokenDetails {
  token: token.TokenResult
  priceFeedId: string
  interest: BigNumber
}
```

Reply:
the above suggestion is to use the already existing structure in  `token.ts`. so you may have to refer to it. 
Also, i noticed that `isLoanToken: boolean` is missing from that one. newly added filed. Could you update that one and use here?